src/App.vue

```diff
@@ -35,7 +35,7 @@ const isPhotoGalleryVisible = computed(() => {
   <main class="h-screen flex">
     <aside class="z-10 absolute">
       <SidePanel v-show="panelStore.isPlanningViewShown === false">
-        <RouterView />
+        <RouterView :key="$route.fullPath" />
```

Comment:
what is this for?

Reply:
I don't know why but without the key on RouteView i had one bug : 
When i clicked on another station on station page, the url change but not the page. With key on RouterView, the url change and the page too.